src/app/appconst.ts

```diff
@@ -47,3 +47,13 @@ export const TabIcons = [
 export const VERSION = __WAVETERM_VERSION__;
 // @ts-ignore
 export const BUILD = __WAVETERM_BUILD__;
+
+/**
+ * Levels for the screen status indicator
+ */
+export enum StatusIndicatorLevel {
```

Comment:
Why does this need to go here? I'd prefer to keep it with the rest of the StatusIndicator code

Reply:
I was previously in types.ts. StatusIndicatorLevel can't be in custom.d.ts cos it's also used at runtime. I was hesitant to move it there because it's not a constant but behaves like a constant.

Reply:
it's fine, let's leave it in appconst